lib/xpm_ruby/connection.rb

```diff
@@ -62,7 +62,7 @@ def handle_response(response)
         when "OK"
           xml["Response"]
         when "ERROR"
-          raise Error.new(response["ErrorDescription"])
+          raise Error.new(xml["Response"]["ErrorDescription"])
```

Comment:
@Subskii just wanted to double check that this change makes sense. The response was a Faraday response which had no ErrorDescription on it, but the xml["Response"] does have one (and at this stage we are guaranteed to have an xml["Response"]).

Reply:
Great catch Tony.

Looks right according to their [docs](https://developer.xero.com/documentation/practice-manager/overview-practice-manager-api). That also explains why I haven't been seeing the error message bubble up in some ailing specs too!

Was surprised the [Staff.list tests](https://github.com/ignitionapp/xpm_ruby/blob/master/spec/xpm_ruby/staff_spec.rb#L29) didn't pick up anything at first but makes sense as we don't assert on the message just class.

Reply:
Ah ok cool thanks!

Reply:
I think we are at the stage now where we should be testing more than just the happy path.
;)